components/Segmented/samples/Segmented.Samples.csproj

```diff
@@ -7,6 +7,8 @@
   <Import Project="$(ToolingDirectory)\ToolkitComponent.SampleProject.props" />
   <ItemGroup>
     <ProjectReference Include="..\..\Extensions\src\CommunityToolkit.WinUI.Extensions.csproj"/>
+    <ProjectReference Include="..\..\Primitives\src\CommunityToolkit.WinUI.Controls.Primitives.csproj"/>
```

Comment:
Need to update to align with https://github.com/CommunityToolkit/Windows/pull/318 before we can close this

Reply:
@Arlodotexe since this is fairly straight-forward of a PR for a new sample, think we could just get this merged and then update in #318?